magneticow/magneticow/magneticow.py

```diff
@@ -67,7 +67,17 @@ def decorated(*args, **kwargs):
 @app.route("/")
 @requires_auth
 def home_page():
-    return flask.render_template("homepage.html")
+    magneticod_db = get_magneticod_db()
+    context = {}
+
+    with magneticod_db:
+        cur = magneticod_db.execute(
+            "SELECT"
+            "    count(id) "
```

Comment:
You should use simply `count()` and let SQLite find the most efficient way possible.

Also, as the query is quite short, you do **not** need to break it into 3 lines.